### PR TITLE
[trivial] parseAttribute() error message did not mention @nogc

### PR DESCRIPTION
**Pull request by Cauterite**
code is self-explanatory

### CODE REVIEW

File: src/parse.d [hunk 1]
```diff
@@ -1384,7 +1384,8 @@ final class Parser : Lexer
             *pudas = UserAttributeDeclaration.concat(*pudas, udas);
         }
         else
-            error("valid attributes are @property, @safe, @trusted, @system, @disable");
+            error("valid attributes are @%s, @%s, @%s, @%s, @%s, @%s",
```
Comment (WalterBright) [inline src/parse.d]: Just have a string, please!

Comment (Cauterite) [inline src/parse.d]: But what's the point of having Idgen if we're just hardcoding the identifiers anyway?

Comment (WalterBright) [inline src/parse.d]: The point is to make them work as identifiers for the lexer.

**Comment by WalterBright** [comment]
thanks!

**Pull request merged**